mne_bids/write.py

```diff
@@ -1190,6 +1190,9 @@ def write_raw_bids(raw, bids_basename, bids_root, events_data=None,
             if ext == '.pdf':
                 bids_fname = op.join(data_path, op.basename(bids_fname))
             _write_raw_fif(raw, bids_fname)
+        elif kind is in ['eeg', 'ieeg'] and ext == '.vhdr':
+            copyfile_brainvision(raw_fname, bids_fname,
```

Comment:
can we limit the changes to function copyfile_brainvision ?

Reply:
Yes, if we pass both `anonymize` and `raw` to `copyfile_brainvision`

We can get rid of
```
elif kind is in ['eeg', 'ieeg'] and ext == '.vhdr':
            copyfile_brainvision(raw_fname, bids_fname,
```

if we do something like

`copyfile_brainvision(raw_fname, bids_fname, anonymize=True/False, date=raw.info['meas_date'])`

Reply:
I made these adjjustments. Do you like it more now?